types/rsvp/rsvp-tests.ts

```diff
@@ -27,6 +29,15 @@ function testAll() {
 
     const anyFailure = RSVP.all([12, 'strings', RSVP.reject('anywhere')]);
     assertType<RSVP.Promise<{}>>(anyFailure);
+
+    let promise1 = RSVP.resolve(1);
+    let promise2 = RSVP.resolve('2');
+    let promise3 = RSVP.resolve({ key: 13 });
+    RSVP.Promise.all([ promise1, promise2, promise3 ], 'my label').then(function(array) {
+        assertType<number>(array[0]);
+        assertType<string>(array[1]);
+        assertType<{ key: number }>(array[2]);
```

Comment:
Huh. Learned something here! I thought they *all* ended up being promises for some reason. This is good to know. Killing my misunderstandings one PR at a time!